### PR TITLE
Add native Elm datepicker blog post

### PR DESCRIPTION
**Pull request by rjdellecese**
All criticism welcome! The more baseless, the better 👌

### CODE REVIEW

**Comment by rjdellecese** [comment]
Ah, need to add an excerpt...

**Comment by rjdellecese** [comment]
Actually, I kinda like the excerpt as it is now, and I think the first paragraph is a good preview of the post itself, but the pattern that we've followed for the other posts has been more of a "post title/one-sentence summary" type of thing. What do you guys think?

**Review by samgqroberts — COMMENTED** [review]
imo keeping the excerpt to one line is good, especially good for the relatively short posts -- ~putting 30% of the entire post in excerpt is :man_shrugging:. Would be curious what your one-line summary of the post would be, too

Also I like the code inlined (though via an iframe? blasphemy!), and I think you could get a little more pop by also inlining some screenshots of what the native datepicker vs. CurrySoftware's datepicker look like. Could help anchor people's understandings of the subjects of this post, especially the visual learners, plus some of the post is about the style itself, so you could point to the screenshot and highlight particular parts of it (eg. "see that in the native one? you can't restyle that")

File: _posts/2019-02-12-native-elm-datepicker.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: "Native Elm Datepicker"
+date: 2019-04-12 5:00:00 -0500
+category: blog
+author: RJ
+author_twitter:
```
Comment (shamshirz) [inline _posts/2019-02-12-native-elm-datepicker.md]: What happens with this? vs. setting as `@sylverstudiosdev`

File: _posts/2019-02-12-native-elm-datepicker.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+image:
```
Comment (shamshirz) [inline _posts/2019-02-12-native-elm-datepicker.md]: Why prefer imageless on this one?

File: _posts/2019-02-12-native-elm-datepicker.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+---
+
+When browser support for date input types was worse (or non-existent), JavaScript datepickers were a common solution if you needed to allow a user to enter a date on a form. These days, browser support is much better, making native browser datepickers a much more viable option. If you need a datepicker in Elm, there is a package that will meet your needs ([CurrySoftware/elm-datepicker](https://github.com/CurrySoftware/elm-datepicker)), but it is more cumbersome than the native browser option, which in Elm is pretty easy to use.
+
+To use the native browser datepicker in Elm all you need to do is add an input tag, capture its input events with `Html.Events.onInput`, and parse the value of the input event for a date. The value will be a string containing a date in the ISO 8601 format (YYYY-MM-DD).
```
Comment (shamshirz) [inline _posts/2019-02-12-native-elm-datepicker.md]: Maybe doc link for `onInput`?

File: _posts/2019-02-12-native-elm-datepicker.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+Here is very simple Ellie example demonstrating how it works:
+
+<iframe src="https://ellie-app.com/embed/3RjFHN9sHzca1" style="width:100%; height:400px; border:0; overflow:hidden;" sandbox="allow-modals allow-forms allow-popups allow-scripts allow-same-origin"></iframe>
```
Comment (shamshirz) [inline _posts/2019-02-12-native-elm-datepicker.md:19]: I hate iframes, but for the purpose of embedding Ellie, I think that is very cool.

**Review by shamshirz — APPROVED** [review]
I love it. I agree with the earlier comments on the excerpt, it stands out a lot, but not a blocker for me.
Excerpt

**Comment by shamshirz** [comment]
Also, twitter link (I'm happy to do this part) looks like this
twitterPicker

(You can test yourself here Additional thought: You can test what it would look like on Twitter here: https://cards-dev.twitter.com/validator)

I find an image and short text more descriptive in general, but I'm still pro merging this.

**Comment by shamshirz** [comment]
@rjdellecese is there anything I can contrive to get this ready? I’d love to release it this weekend. I’ve got 10 hours in the car!

File: _posts/2019-02-12-native-elm-datepicker.md (hunk 4 above)
Comment (rjdellecese) [reply _posts/2019-02-12-native-elm-datepicker.md:19]: Tis the only way that Ellie offers

File: _posts/2019-02-12-native-elm-datepicker.md (hunk 2 above)
Comment (rjdellecese) [reply _posts/2019-02-12-native-elm-datepicker.md]: Should rename this field `twitter_image`

**Pull request merged**